### PR TITLE
Add 6.9.0 CHANGELOG

### PR DESCRIPTION
**Pull request by SarahFrench**
paired with https://github.com/hashicorp/terraform-provider-google/pull/20013

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+* looker: added `deletion_policy` to `google_looker_instance` to allow force-destroying instances with nested resources by setting `deletion_policy = FORCE` ([#8453](https://github.com/hashicorp/terraform-provider-google-beta/pull/8453))
+* monitoring: added `alert_strategy.notification_prompts` field to `google_monitoring_alert_policy` ([#8457](https://github.com/hashicorp/terraform-provider-google-beta/pull/8457))
+* sql: removed the client-side default of `ENTERPRISE` for `edition` in `google_sql_database_instance` so that `edition` is determined by the API when unset. This will cause new instances to use `ENTERPRISE_PLUS` as the default for POSTGRES_16. ([#8490](https://github.com/hashicorp/terraform-provider-google-beta/pull/8490))
+* vmwareengine: added `autoscaling_settings` to `google_vmwareengine_cluster` resource ([#8477](https://github.com/hashicorp/terraform-provider-google-beta/pull/8477))
+* workstations: added `maxUsableWorkstations`  field to `WorkstationConfiguration` resource. ([#8421](https://github.com/hashicorp/terraform-provider-google-beta/pull/8421))
```
Comment (SarahFrench) [inline CHANGELOG.md:29]:
```suggestion
* workstations: added `max_usable_workstations`  field to `google_workstations_workstation_config` resource. ([#8421](https://github.com/hashicorp/terraform-provider-google-beta/pull/8421))
```

**Review by SarahFrench — COMMENTED** [review]
First pass review

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+* monitoring: added `alert_strategy.notification_prompts` field to `google_monitoring_alert_policy` ([#8457](https://github.com/hashicorp/terraform-provider-google-beta/pull/8457))
+* storage: added 'hierarchical_namespace' to 'google_storage_bucket' resource ([#8428](https://github.com/hashicorp/terraform-provider-google-beta/pull/8428))
+* sql: removed the client-side default of `ENTERPRISE` for `edition` in `google_sql_database_instance` so that `edition` is determined by the API when unset. This will cause new instances to use `ENTERPRISE_PLUS` as the default for POSTGRES_16. ([#8490](https://github.com/hashicorp/terraform-provider-google-beta/pull/8490))
+* vmwareengine: added `autoscaling_settings` to `google_vmwareengine_cluster` resource ([#8477](https://github.com/hashicorp/terraform-provider-google-beta/pull/8477))
+* workstations: added `max_usable_workstations`  field to `google_workstations_workstation_config` resource. ([#8421](https://github.com/hashicorp/terraform-provider-google-beta/pull/8421))
```
Comment (SarahFrench) [inline CHANGELOG.md:28]:
```suggestion
* workstations: added `max_usable_workstations` field to `google_workstations_workstation_config` resource. ([#8421](https://github.com/hashicorp/terraform-provider-google-beta/pull/8421))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
 ## 6.9.0 (Unreleased)
 
+NOTES:
+* provider: minor refactoring of how we run acceptance tests in VCR mode ([#8425](https://github.com/hashicorp/terraform-provider-google-beta/pull/8425))
+
```
Comment (SarahFrench) [inline CHANGELOG.md:5]: We might want to remove this, but figured I'd flag for others to give their opinion on.

```suggestion
```

**Review by trodge — APPROVED** [review]

**Pull request merged**